include/rpm/rpmpgp.h

```diff
@@ -1113,6 +1119,95 @@ int pgpDigParamsCmp(pgpDigParams p1, pgpDigParams p2);
  */
 unsigned int pgpDigParamsAlgo(pgpDigParams digp, unsigned int algotype);
 
+/** \ingroup rpmpgp
+ * Returns the issuer or the object's Key ID.
+ *
+ * If the object is a signature, then this returns the Key ID stored in the
+ * first Issuer or Issuer Fingerprint subpacket as a hex string.
```

Comment:
Just Issuer, actually.  RPM doesn’t currently handle issuer fingerprint subpackets at all (though I have a PR for that).

Reply:
The comment is technically correct ("or") :rofl: .  The comment came from the Sequoia port, which does handle fingerprints.  But I'll update the comment for this patch, thanks.